k8s/apply.sh.tpl

```diff
@@ -24,4 +24,4 @@ function guess_runfiles() {
 RUNFILES="${PYTHON_RUNFILES:-$(guess_runfiles)}"
 
 PYTHON_RUNFILES=${RUNFILES} %{resolve_script} | \
-  kubectl --cluster="%{cluster}" --context="%{context}" --user="%{user}" %{namespace_arg} apply -f -
\ No newline at end of file
+  kubectl --cluster="%{cluster}" --context="%{context}" --user="%{user}" %{namespace_arg} $@ apply -f -
```

Comment:
@omadawn Any reason you put this `$@` before the `apply`? I was trying to run `bazel-bin/examples/hellohttp2/py/staging.apply --dry-run` and the way this currently is setup this does not work. If the `$@` is moved after the `apply` then the dry-run flag does indeed work. Would be nice to get #145 fixed.

Reply:
That won't actually work.
The problem is that flags like --dry-run are not global. It works for apply but not for say describe. With the current architecture of the plugin all flags are applied to all options. We would need to provide a way to specify individual settings for different activities.

Reply:
But do the global flags not also work behind the `apply/describe` command? Looking at the test `kubectl apply/describe --v=0 --alsologtostderr=true --stderrthreshold=1` seems to work for me also.